core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/TestCASLockFactory.java

```diff
@@ -375,13 +380,103 @@ public boolean getRemoteRouting()
                             {
                                 return true;
                             }
+
+                            @Override
+                            public String getSerialConsistency(){
+                                return "DEFAULT";
+                            }
                         })
                         .withHostStates(hostStates)
                         .withStatementDecorator(s -> s)
                         .withKeyspaceName(myKeyspaceName)
                         .build());
     }
 
+    @Test
+    public void testRemoteRoutingTrueWithDefaultSerialConsistency() {
```

Comment:
Incorrect formatting {} should be placed on newlines.

Reply:
Done